src/main/kotlin/no/skatteetaten/aurora/gobo/graphql/GoboDataLoader.kt

```diff
@@ -26,7 +27,7 @@ abstract class GoboDataLoader<K, V> : KotlinDataLoader<K, V> {
                 @OptIn(DelicateCoroutinesApi::class)
                 GlobalScope.async(Dispatchers.IO + MDCContext() + TracingContextElement()) {
                     getByKeys(keys, env.graphqlContext)
-                }.asCompletableFuture()
+                }.asCompletableFuture().orTimeout(3, TimeUnit.MINUTES)
```

Comment:
Denne timeouten burde vel være konfigurerbar?

Reply:
Tja, det er noe vi kan diskutere. Disse 2 timeoutsene på CompletableFuture er kun en siste utvei for å unngå at de henger. De er satt såpass høyt fordi dette skal egentlig ikke skje.
Tror ikke det vil være aktuelt å justere så mye på disse underveis, men klart vi kan snakke om hva verdien bør være.

Reply:
Den er jo kortere enn HA-proxy timeouten (som vel er 5 minutter?), så den vil jo potensielt kunne skje?

Reply:
Vi kan nok øke den til haproxy timeouten ja, det høres ut som et greit utgangspunkt syns jeg

Reply:
Så da øker jeg den til 5 min? Skal jeg gjøre den konfigurerbar?